app/globals.css

```
@@ -235,6 +246,21 @@
   }
 }
 
+/* Lightbox media fade */
+@keyframes lbFadeIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "lbFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)